src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,459 @@
+  sql_file="hints_adoption_trends.sql"
+  )
+}}
+
+### By Rank
+
+{{ figure_markup(
+  image="rel-preload-adoption-by-rank.png",
+  caption="Adoption of rel=\"preload\" segmented by CrUX rank.",
+  description="Bar chart showing the adoption of rel=\"preload\" segmented by CrUX rank. rel=\"preload\" has a 44% for the top 1,000 sites on mobile and 44% on desktop. rel=\"preload\" has a 43% for the top 10,000 sites on mobile and 44% on desktop. rel=\"preload\" has a 35% for the top 100,000 sites on mobile and 36% on desktop. rel=\"preload\" has a 27% for the top 1 million sites on mobile and 27% on desktop. rel=\"preload\" has a 22% for the top 10 million sites on mobile and 22% on desktop.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTgxCYoH2vDbhTky1qQKEeV034kReHF8JYYq0aDyoo6LG22XL6Tar3dNPsqU1_zyvR6vuFEdMGoF1oP/pubchart?oid=291501434&format=interactive",
+  sheets_gid="1880502987",
+  sql_file="hints_adoption_by_rank.sql"
+  )
+}}
+
+You can observe that when segmenting the data by [rank](./methodology#chrome-ux-report), the adoption rates change notably, with the `preload` hint increasing from 22.10% unranked, to claim the top spot with an adoption rate of 44.28% amongst the top 1,000 sites.
+
+{{ figure_markup(
+  image="rel-dns-prefetch-adoption-by-rank.png",
+  caption="Adoption of rel=\"dns-prefetch\" segmented by CrUX rank.",
+  description="Bar chart showing the adoption of rel=\"dns-prefetch\" segmented by CrUX rank. rel=\"dns-prefetch\" has a 29% for the top 1,000 sites on mobile and 28% on desktop. rel=\"dns-prefetch\" has a 25% for the top 10,000 sites on mobile and 25% on desktop. rel=\"dns-prefetch\" has a 23% for the top 100,000 sites on mobile and 23% on desktop. rel=\"dns-prefetch\" has a 27% for the top 1 million sites on mobile and 27% on desktop. rel=\"dns-prefetch\" has a 36% for the top 10 million sites on mobile and 36% on desktop.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTgxCYoH2vDbhTky1qQKEeV034kReHF8JYYq0aDyoo6LG22XL6Tar3dNPsqU1_zyvR6vuFEdMGoF1oP/pubchart?oid=1340595902&format=interactive",
+  sheets_gid="1880502987",
+  sql_file="hints_adoption_by_rank.sql"
+  )
+}}
+
+`dns-prefetch` is the only resource hint which exhibits a decrease in adoption when comparing the top 1,000 sites with the overall adoption.
+
+{{ figure_markup(
+  image="rel-preconnect-adoption-by-rank.png",
+  caption="Adoption of rel=\"preconnect\" segmented by CrUX rank.",
+  description="Bar chart showing the adoption of rel=\"preconnect\" segmented by CrUX rank. rel=\"preconnect\" has a 30% for the top 1,000 sites on mobile and 29% on desktop. rel=\"preconnect\" has a 25% for the top 10,000 sites on mobile and 25% on desktop. rel=\"preconnect\" has a 18% for the top 100,000 sites on mobile and 18% on desktop. rel=\"preconnect\" has a 14% for the top 1 million sites on mobile and 14% on desktop. rel=\"preconnect\" has a 13% for the top 10 million sites on mobile and 13% on desktop.",
```

Comment:
Code format. Can also switch to single quotes to avoid having to escape:

```suggestion
  caption='Adoption of `rel="preconnect"` segmented by CrUX rank.',
  description='Bar chart showing the adoption of `rel="preconnect"` segmented by CrUX rank. `rel="preconnect"` has a 30% for the top 1,000 sites on mobile and 29% on desktop. `rel="preconnect"` has a 25% for the top 10,000 sites on mobile and 25% on desktop. `rel="preconnect"` has a 18% for the top 100,000 sites on mobile and 18% on desktop. `rel="preconnect"` has a 14% for the top 1 million sites on mobile and 14% on desktop. `rel="preconnect"` has a 13% for the top 10 million sites on mobile and 13% on desktop.',
```